Tasks/Maven/pmdForMaven.ts

```diff
@@ -1,9 +1,54 @@
 /// <reference path='../../definitions/vsts-task-lib.d.ts' />
 
+import util = require('util');
+import path = require('path');
+import fs = require('fs');
+import xml2js = require('xml2js');
```

Comment:
Accepted, in progress.

Reference: https://github.com/Microsoft/vso-agent-tasks/commit/11707b38f25d7110b1e5036b014a310eee02eabd